src/NuGet.Core/NuGet.Packaging/Telemetry/PackageExtractionTelemetryEvent.cs

```diff
@@ -32,7 +33,8 @@ public class PackageExtractionTelemetryEvent : TelemetryEvent
                     { nameof(PackageSaveMode), packageSaveMode }
                 })
         {
-            AddPiiData(nameof(PackageId), packageId.ToString());
```

Comment:
We should `ToLowerInvariant` both fields. 

`ToString` on ID is redundant right?

Unit test on telemetry key names or make into a dedicated constant. Right now a refactor could change telemetry schema.

Reply:
I'll update `id` to `ToLowerInvariant`

I agree that key names should be dedicated constant and I see through out the telemetry events, it has been converted like that. I've logged a tracking issue to fix it across all events - https://github.com/NuGet/Home/issues/6683

Reply:
As part of this PR we should `ToLowerInvariant` the version as well. This value is also case-insensitive in the same way as ID.

Reply:
done! I was under the impression that `ToNoumalizeString` will take care of that.